src/Adapters/ADTAdapter.ts

```diff
@@ -583,7 +583,9 @@ export default class ADTAdapter implements IADTAdapter {
                     return null;
                 }
             });
-            return new ADTModelsData(axiosResult?.data);
+            return new ADTModelsData(
+                (axiosResult?.data as unknown) as IADTModel[]
```

Comment:
why is this needed now?

Reply:
there was a type clash in axiosResult; where its trying to assign values of DTModel[] to IADTModel[]

Reply:
I'm confused why this change is introducing that issue. Are you sure you don't need to go down another layer with data.data? That seemed to happen in the other places you were using axios results.

Reply:
layering doesn't seems to be an issue here! with different underlying shape, I checked the upstream and don't see any specific cast being made to axiosResult